tests/test_debugging.py

```diff
@@ -717,12 +717,14 @@ def circuit():
 
         # Make sure shots are overridden correctly
         result = circuit(shots=200)
-        assert np.allclose(
-            result[0],
-            np.array([1 / 3, 0.0, 0.0, 1 / 3, 0.0, 0.0, 1 / 3, 0.0, 0.0]),
-            atol=0.1,
-            rtol=0,
-        )
+
+        # !Note: the following asserting used to cause lots of flaky failures
+        # because actually we don't even need to check the accuracy of the
+        # probs output but it only used ~100 order shots super insufficient
+        # we removed this in PR#7341 but maybe the shape comparison is not
+        # enough to say it's "overridden correctly"? Anyways, it's hard to
+        # imagine how would the overridability be broken in the first place...
```

Comment:
Would shorten this explanation, no need to explain the details of the flaky test being removed.
```suggestion
        # Note: We do not need to check the accuracy of the output,
        # but only that multiple snapshots are returned and shots are
        # correctly overridden.
```